dev_tools/scripts/es_test.sh

```diff
@@ -2,22 +2,28 @@
 
 NOW=$(date '+%Y%m%d%H%M')
 
-CONFIGFILE="dev_tools/config.yaml" go run . --action es_create --name "$NOW-123"
-CONFIGFILE="dev_tools/config.yaml" go run . --action es_create --name "abc-$NOW"
+for i in {1..100}; do
```

Comment:
Possibly put the number of documents to use for test in a variable ("making configurable"), and this would then become `for i in $(seq 1 $variable); do`.